playbooks/pmm2-client_integration.yaml

```diff
@@ -0,0 +1,149 @@
+    when: lookup('env', 'install_repo') == "main"
+
+  - name: install PMM client new deb packages
+    apt: name={{item}} update_cache=yes state=latest
+    with_items:
+    - pmm2-client
+    when: ansible_os_family == "Debian"
+
+  - name: install PMM client new rpm packages
+    yum: name={{item}} state=latest
+    with_items:
+    - pmm2-client
+    when: ansible_os_family == "RedHat"
+
+  - name: check that PMM client version is correct
+    command: /package-testing/version_check.sh pmm2
+
+  - name: configure pmm client to connect to server
+    shell: pmm-agent setup --server-address={{ pmm_server_address }}:443 --server-insecure-tls --server-username=admin --server-password=admin $(ip addr show|grep "inet 10."|grep -o "10.*/"|sed 's:/::')
+
+### start PS
+  - name: install Percona Server new deb packages
+    apt: name={{item}} update_cache=no state=latest
+    with_items:
+    - percona-server-server-5.7
+    - percona-server-test-5.7
+    - percona-server-5.7-dbg
+    - percona-server-source-5.7
+    - percona-server-tokudb-5.7
+    when: ansible_os_family == "Debian"
+
+  - name: install Percona Server new rpm packages
+    yum: name={{item}} state=latest
+    with_items:
+    - Percona-Server-server-57
+    - Percona-Server-test-57
+    - Percona-Server-57-debuginfo
+    - Percona-Server-devel-57
+    - Percona-Server-tokudb-57
+    - Percona-Server-shared-57
+    - Percona-Server-client-57
+    when: ansible_os_family == "RedHat"
+
+  - name: start mysql service with service command
+    command: service mysql start
+    when: ansible_os_family == "RedHat" and ansible_distribution_major_version == "6"
+
+  - name: start mysql service with systemctl command
+    command: systemctl start mysql
+    when: ansible_os_family == "RedHat" and ansible_distribution_major_version == "7"
+
+  - name: set root password on centos
+    command: /package-testing/setpass_57.sh
+    when: ansible_os_family == "RedHat"
+
+  - name: copy .my.cnf with credentials on centos
+    template: src=../templates/my_57.j2
+              dest=/root/.my.cnf
+              mode=0640 owner=root group=root
+    when: ansible_os_family == "RedHat"
+
+  - name: add mysql metrics and queries to monitoring
+    command: /package-testing/scripts/pmm2_add_instance.sh mysql
+
+  - name: check if mysql metrics and queries are running
+    command: /package-testing/scripts/pmm2_service_check.sh mysql
+### end PS
+
+### start PSMDB
+  - name: install Percona Server for MongoDB 3.4 new deb packages
+    apt: name={{item}} update_cache=yes state=latest
+    with_items:
+    - percona-server-mongodb-34
+    - percona-server-mongodb-34-dbg
+    when: ansible_os_family == "Debian" and ansible_distribution_release != "wheezy"
+
+  - name: install Percona Server for MongoDB 3.4 new rpm packages
+    yum: name={{item}} state=latest
+    with_items:
+    - Percona-Server-MongoDB-34
+    - Percona-Server-MongoDB-34-debuginfo
+    when: ansible_os_family == "RedHat"
```

Comment:
Same as above, probably better to include this: https://github.com/Percona-QA/package-testing/blob/master/tasks/install_psmdb_34.yml